docs/content/Getting-Started/Core/02-Create-a-project.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Create a project
+permalink: /getting-started/core/create-a-project
+category: Getting Started
+subCategory: Core
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
+## Scaffold a project
+
+Start by opening your terminal to create a new folder for the project, then
+create a `docker-compose.yml` file within it:
+
+```bash{promptUser: user}
+mkdir my-first-cube-project && cd $_
+touch docker-compose.yml
+```
+
+Open the `docker-compose.yml` file and add the following content:
```

Comment:
What do you think about using this [`cat EOF`](https://linuxhint.com/what-is-cat-eof-bash-script/) technique and compressing it to a single block of commands?

Reply:
I tried it:

<img width="885" alt="image" src="https://user-images.githubusercontent.com/1781985/229145877-3e4cb766-01b0-406c-b2de-2020c8bac3a4.png">

IMO the syntax highlighting looks better when its a separate snippet